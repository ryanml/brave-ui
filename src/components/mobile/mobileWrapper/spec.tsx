/* global jest, expect, describe, it, afterEach */
import * as React from 'react'
import { shallow } from 'enzyme'
import { create } from 'react-test-renderer'
import MobileWrapper from './index'
import { TestThemeProvider } from '../../../theme'

describe('Modal tests', () => {
  const baseComponent = (props?: object) => <TestThemeProvider><MobileWrapper id='mobile-wrapper' {...props} /></MobileWrapper>

  describe('basic tests', () => {
    it('matches the snapshot', () => {
      const component = baseComponent()
      const tree = create(component).toJSON()
      expect(tree).toMatchSnapshot()
    })

    it('renders the component', () => {
      const wrapper = shallow(baseComponent())
      const assertion = wrapper.find('#mobile-wrapper').length
      expect(assertion).toBe(1)
    })
  })
})
