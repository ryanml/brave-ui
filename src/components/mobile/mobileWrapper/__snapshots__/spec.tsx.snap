// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal tests basic tests matches the snapshot 1`] = `
.c0 {
  display: block;
  width: 360px;
  height: 640px;
}

<div
  className="c0"
  id="mobile-wrapper"
/>
`;
